config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.ttl

```diff
@@ -485,9 +490,464 @@ ext:ProjectfaseringMetProjectverloopTimingEnRisicobeheerProjectGroup
+     
+     
+
+    ##########################################################
+    # Field: Projectonderdeel naam Input
+    ##########################################################
+
+    ext:projectOnderdeelNaamF
+        a                 form:Field ;
+        sh:name           "Projectnaam" ;
+        sh:order          1 ;
+        sh:path           lblodSubsidie:projectOnderdeelNaam ;
+        form:validations
+                          [ a                 form:MaxLength ;
+                            form:grouping     form:MatchEvery ;
+                            form:max          "150" ;
+                            sh:resultMessage  "Max. karakters overschreden." ;
+                            sh:path            lblodSubsidie:projectOnderdeelNaam
+                          ],
+                          [ a                 form:RequiredConstraint ;
+                            form:grouping     form:Bag ;
+                            sh:resultMessage  "Dit veld is verplicht." ;
+                            sh:path           lblodSubsidie:projectOnderdeelNaam            
+                          ] ;
+        form:displayType  displayTypes:defaultInput ;
+        sh:group          ext:projectOnderdeelPg-sub .
+
+
+    ##########################################################
+    # Field: Projectonderdeel toelichting Input
+    ##########################################################
+
+    ext:projectOnderdeelBeschrijvingF
+        a                 form:Field ;
+        sh:name           "Geef een korte toelichting over wat dit projectonderdeel inhoudt" ;
+        sh:order          1 ;
+        sh:path           lblodSubsidie:projectOnderdeelBeschrijving ;
+        form:validations
+                          [ a                 form:MaxLength ;
+                            form:grouping     form:MatchEvery ;
+                            form:max          "750" ;
+                            sh:resultMessage  "Max. karakters overschreden." ;
+                            sh:path            lblodSubsidie:projectOnderdeelBeschrijving
+                          ],
+                          [ a                 form:RequiredConstraint ;
+                            form:grouping     form:Bag ;
+                            sh:resultMessage  "Dit veld is verplicht." ;
+                            sh:path           lblodSubsidie:projectOnderdeelBeschrijving            
+                          ] ;
+        form:displayType  displayTypes:textArea ;
+        sh:group          ext:projectOnderdeelPg-sub .
+
+
+    ##########################################################
+    # Listing: Projectonderdeel Timing
+    ##########################################################
+        ##########################################################
+        #  property-group
+        ##########################################################
+        ext:projectOnderdeelTimingListingPg a form:PropertyGroup;
```

Comment:
The propertyGroup also supports configuring the header by passing a form:options with level and skin properties. We should probably revisit after all the sections are added to make sure things look more as expected.

More info: https://github.com/lblod/ember-submission-form-fields/pull/96

Reply:
Ah cool, didnt know we could do that. Will cleanup styles in next iteration :)